spec/requests/renews_spec.rb

```diff
@@ -1,9 +1,19 @@
 # frozen_string_literal: true
 
 require "rails_helper"
+require "defra_ruby_companies_house"
 
 RSpec.describe "Renews", type: :request do
   let(:registration) { create(:registration) }
+  let(:transient_registration_token) { WasteExemptionsEngine::RenewingRegistration.last.token }
+  let(:company_name) { Faker::Company.name }
```

Comment:
I thunk lines 9-16 belong in the "when the business type is a company or llp" context block. Are those values used anywhere else?

Reply:
They are used during the check for the user as well. Have changed them though to be inside that block